backend/database/migrations/2023_02_09_051646_taken_categories.php

```diff
@@ -13,15 +13,17 @@
      */
     public function up()
     {
-        Schema::create('learned_words', function (Blueprint $table) {
+        Schema::create('taken_categories', function (Blueprint $table) {
```

Comment:
Why are you renaming this table from `learned_words` to `learned_words`?
I guess it should be new migration